### PR TITLE
XRT-410 fix xrt build failure and update petalinux verson

### PR DESCRIPTION
**Pull request by Himanshu-xilinx**
* update petalinux build version
* build rpm with generic machine

Tested for zynqmp, zynq and versal

### CODE REVIEW

File: build/build_edge.sh [hunk 1]
```diff
@@ -40,6 +40,7 @@ install_recipes()
         echo "EXTERNALSRC = \"$XRT_REPO_DIR/src\"" >> $XRT_BB
         echo 'EXTERNALSRC_BUILD = "${WORKDIR}/build"' >> $XRT_BB
         echo 'PACKAGE_CLASSES = "package_rpm"' >> $XRT_BB
+        echo 'INSANE_SKIP_${PN}-dev += "dev-elf"' >> $XRT_BB
```
Comment (mamin506) [inline build/build_edge.sh:43]: #3176 has been merged. This line is not necessary.
BTW, #3187 do cleanup the libraries install path. It is final solution to fix the QA issue.

Comment (chvamshi-xilinx) [reply build/build_edge.sh:43]: @Himanshu-xilinx , As the original issue is fixed, please enable the check again.

Comment (Himanshu-xilinx) [reply build/build_edge.sh]: updated

**Comment by maxzhen** [comment]
Please use single space b/w task number and synopsis, not ":". Use same standard msg in both PR title and commit msg, please.

**Comment by chvamshi-xilinx** [comment]
retest this please

**Pull request merged**